pkg/controller/signup.go

```diff
@@ -45,7 +46,11 @@ func (s *Signup) PostHandler(ctx *gin.Context) {
 		return
 	}
 
-	log.Infof(ctx, "UserSignup %s created", userSignup.Name)
+	if _, exists := userSignup.Annotations[toolchainv1alpha1.UserSignupActivationCounterAnnotationKey]; exists {
+		log.Infof(ctx, "UserSignup created: %s", userSignup.Name)
```

Comment:
Shouldn't this be the other way around? The UserSignup is created only when the annotation does not exist?
```suggestion
	if _, exists := userSignup.Annotations[toolchainv1alpha1.UserSignupActivationCounterAnnotationKey]; !exists {
		log.Infof(ctx, "UserSignup created: %s", userSignup.Name)
```

Reply:
ah right! 🤦‍♂️